test/foo/lib/config/config.yaml

```diff
@@ -0,0 +1,3 @@
+VAR0: fizz
```

Comment:
We can name this file with test, ex. `test_config.yml`
What do you think? It's placed in tests, but anyway?